certified-connectors/Data8/apiDefinition.swagger.json

```diff
@@ -1,362 +1,818 @@
                   },
-                  "TelephoneNumber": {
-                    "type": "string",
-                    "description": "Returns the telephone number if callable, null if not.",
-                    "title": "Telephone Number"
-                  }
+                  "description": "Address"
+                },
+                "AcceptsBACSPayments": {
+                  "type": "boolean",
+                  "description": "Indicates if the bank branch can accept payments made through the BACS system.",
+                  "title": "Accepts BACS Payments"
+                },
+                "AcceptsDirectDebitTransactions": {
+                  "type": "boolean",
+                  "description": "Indicates if the bank branch can accept direct debit payments.",
+                  "title": "Accepts Direct Debit Transactions"
+                },
+                "AcceptsDirectCreditTransactions": {
+                  "type": "boolean",
+                  "description": "Indicates if the bank branch can accept direct credit payments.",
+                  "title": "Accepts Direct Credit Transactions"
+                },
+                "AcceptsUnpaidChequeClaimTransactions": {
+                  "type": "boolean",
+                  "description": "Indicates if the bank branch can accept claims for unpaid cheques.",
+                  "title": "Accepts Unpaid Cheque Claim Transactions"
+                },
+                "AcceptsBuildingSocietyCreditTransactions": {
+                  "type": "boolean",
+                  "description": "Indicates if the bank branch can accept credits from building societies.",
+                  "title": "Accepts Building Society Credit Transactions"
+                },
+                "AcceptsDividendInterestPaymentTransactions": {
+                  "type": "boolean",
+                  "description": "Indicates if the bank branch can accept dividend interest payments.",
+                  "title": "Accepts Dividend Interest Payment Transactions"
+                },
+                "AcceptsDirectDebitInstructionTransactions": {
+                  "type": "boolean",
+                  "description": "Indicates if the bank branch can accept direct debit instructions.",
+                  "title": "Accepts Direct Debit Instruction Transactions"
+                },
+                "AcceptsCHAPSPayments": {
+                  "type": "boolean",
+                  "description": "Indicates if the bank branch can accept payments made through the CHAPS system.",
+                  "title": "Accepts CHAPS Payments"
+                },
+                "AcceptsCheques": {
+                  "type": "boolean",
+                  "description": "Indicates if the bank branch can accept payments made by cheque.",
+                  "title": "Accepts Cheques"
+                },
+                "AcceptsFasterPayments": {
+                  "type": "boolean",
+                  "description": "",
```

Comment:
@mattybeard Empty description can be removed, or you can provide a valid description value.